letsencrypt/cli.py

```diff
@@ -1583,6 +1584,10 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):
         "security", "--rsa-key-size", type=int, metavar="N",
         default=flag_default("rsa_key_size"), help=config_help("rsa_key_size"))
     helpful.add(
+        "security", "--ecdsa_curve", default=flag_default("ecdsa_curve"))
```

Comment:
Help text needs to be added indeed. Good point about the switch style.

For now, the options are hardcoded, but it should be configurable methinks.